tests/foreman/cli/test_subscription.py

```diff
@@ -135,15 +124,18 @@ def test_positive_manifest_history(function_org, manifest_clone_upload):
 
     :CaseImportance: Medium
     """
-    Subscription.list({'organization-id': function_org.id}, per_page=None)
-    history = Subscription.manifest_history({'organization-id': function_org.id})
-    assert f'{function_org.name} file imported successfully.' in ''.join(history)
+    Subscription.list({'organization-id': function_entitlement_manifest_org.id}, per_page=None)
+    history = Subscription.manifest_history(
+        {'organization-id': function_entitlement_manifest_org.id}
+    )
+    assert f'{function_entitlement_manifest_org.name} file imported successfully.' in ''.join(
+        history
+    )
```

Comment:
```suggestion
    org = function_entitlement_manifest_org
    Subscription.list({'organization-id': org.id}, per_page=None)
    history = Subscription.manifest_history({'organization-id': org.id})
    assert f'{org.name} file imported successfully.' in ''.join(history)
```

Reply:
Good suggestion, your version is significantly more readable.